community/basic_examples/hw_aware_synthesis.ipynb

```diff
@@ -0,0 +1,601 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        backend_service_provider="Classiq", #Feel free to change this to any provider classiq supports (IBM Quantum. Amazon Bracket, INOQ, etc)
</pre><p>It is either you use the <code>custom_hardawre_settings</code>,or you specify the backend.</p><p>In this example, you specify explicitly the connectivity and basis gates so there is no need to specify the <code>backend_service_provider</code> and <code>backend_name</code> </p><p>So Please remove them from the code and explanations.</p><p>See more details here:</p><p><a href="https://docs.classiq.io/latest/reference-manual/platform/synthesis/hardware-aware-synthesis#example" target="_blank">https://docs.classiq.io/latest/reference-manual/platform/synthesis/hardware-aware-synthesis/#example</a>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>